charts/stable/valheim/questions.yaml

```diff
@@ -9,205 +9,201 @@ questions:
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
   - variable: secretEnv
-    group: "App Configuration"
-    label: "Image Secrets"
+    group: App Configuration
+    label: Image Secrets
     schema:
       additional_attrs: true
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: "SUPERVISOR_HTTP_USER"
+          label: SUPERVISOR_HTTP_USER
           schema:
             type: string
-            default: "admin"
+            default: admin
             required: true
         - variable: SUPERVISOR_HTTP_PASS
-          label: "SUPERVISOR_HTTP_PASS"
+          label: SUPERVISOR_HTTP_PASS
           schema:
             type: string
             required: true
             private: true
-            default: "REPLACETHIS"
+            default: REPLACETHIS
         - variable: SERVER_PASS
-          label: "SERVER_PASS"
+          label: SERVER_PASS
           schema:
             type: string
             required: true
             private: true
-            default: "REPLACETHIS"
+            default: REPLACETHIS
   - variable: env
-    group: "App Configuration"
-    label: "Image Environment"
+    group: App Configuration
+    label: Image Environment
     schema:
       additional_attrs: true
       type: dict
       attrs:
         - variable: STATUS_HTTP
-          label: "STATUS_HTTP"
+          label: STATUS_HTTP
+          description: Turn on the status http server. Only useful on public servers (SERVER_PUBLIC=true).
           schema:
             type: boolean
             default: true
         - variable: SUPERVISOR_HTTP
-          label: "SUPERVISOR_HTTP"
+          label: SUPERVISOR_HTTP
+          description: Turn on supervisor's http server.
           schema:
             type: boolean
             default: true
```

Comment:
Both of those can go I guess.
Unless there is a reason someone would need to disable one of them